lib/MetaLoader.php

```diff
@@ -531,7 +543,6 @@ public function writeMetadataSerialize(string $outputDir): void
     private function getTime(): string
     {
         // The current date, as a string
-        date_default_timezone_set('UTC');
-        return date('Y-m-d\\TH:i:s\\Z');
+        return gmdate('Y-m-d\\TH:i:s\\Z');
```

Comment:
Good change, not strictly related to the PR but since it's in a separate commit we can probably rebase and merge and we'll be fine.